### PR TITLE
Add vim-polyglot to plugins

### PR DESCRIPTION
**Pull request by tsigo**
Found this neat plugin the other day that includes many language-specific Vim
plugins so you don't have to add them individually as you start playing with a
new language.

[Check out the full list in the README](https://github.com/sheerun/vim-polyglot#language-packs).

### CODE REVIEW

File: vimrc [hunk 1]
```diff
@@ -39,10 +39,6 @@ augroup vimrcEx
     \   exe "normal g`\"" |
     \ endif
 
-  " Set syntax highlighting for specific file types
-  autocmd BufRead,BufNewFile Appraisals set filetype=ruby
```
Comment (tsigo) [inline vimrc]: https://github.com/sheerun/vim-polyglot/blob/master/ftdetect/polyglot.vim#L178

File: vimrc [hunk 2]
```diff
@@ -39,10 +39,6 @@ augroup vimrcEx
...
-  autocmd BufRead,BufNewFile *.md set filetype=markdown
```
Comment (tsigo) [inline vimrc]: https://github.com/sheerun/vim-polyglot/blob/master/ftdetect/polyglot.vim#L102-L107

**Comment by Keithbsmiley** [comment]
This plugin is super heavy weight. Personally I'd really rather not pull it in.

**Comment by Keithbsmiley** [comment]
Also many syntax files aren’t kept up to date. See https://github.com/sheerun/rspec.vim Which is a fork of my repo which hasn't pulled in changes for over a year.

**Comment by gylaz** [comment]
I agree with @Keithbsmiley. Seems too heavy. Also, we haven't had much churn around adding new languages.

**Pull request closed without merging**